.circleci/config.yml

```diff
@@ -43,9 +43,9 @@ jobs:
             - code-dot-org-source-v7-
 
       - checkout
-      - run:
-          name: merge staging
-          command: bin/circle/merge-base
+    #- run:
+        #name: merge staging
+        #command: bin/circle/merge-base
```

Comment:
revert more temporary changes here?

Reply:
I will revert this one, but it's needed until the circleci tests pass.

Reply:
Reverted (circle ci tests on this branch will fail due to this, but should succeed on staging)